src/config.ts

```diff
@@ -18,6 +18,7 @@ type Config = {
     /* eslint-disable @typescript-eslint/no-explicit-any */
     katexOptions?: any;
     tocOptions?: any;
+    alertsOptions?: any;
     /* eslint-enable @typescript-eslint/no-explicit-any */
```

Comment:
Just saw that this wasn't typed while looking at the lower case thing, we should definitely fix that. From the `customization.md` I read:

```suggestion
    /* eslint-enable @typescript-eslint/no-explicit-any */
    alertsOptions?: {
        icons?: Record<string, string>;
        titles?: Record<string, string>;
        fallbackIcon?: string;
    };
```

Is that right?

(This differs from `katexOptions` and `tocOptions` because those are plugins that don't expose the correct types, that's why we have `any` there. For our own code we should type properly)

Reply:
Thanks, was due to inexperience with TS :)

Reply:
I'll do this along with naming change `=> alertOptions`